client/src/app/transactions/transactions.component.html

```diff
@@ -15,8 +15,11 @@
   </ng-container>
   <ng-container matColumnDef="actions">
     <th mat-header-cell *matHeaderCellDef>Actions</th>
-    <td mat-cell *matCellDef="let element"><button mat-button [routerLink]="['./edit/', element.id]">EDIT</button></td>
+    <td mat-cell *matCellDef="let element">
+      <button mat-button [routerLink]="['./edit/', element.id]">EDIT</button>
+      <button mat-button (click)="onDelete(element.id)">DELETE</button>
+    </td>
   </ng-container>
   <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
   <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
-</table>
+</table>
```

Comment:
The only problem I could find was this missing line break at the end of this file. Otherwise, it looks good. 

I especially liked that you didn't try to complicate the issue. You accomplished exactly what was described in the issue with minimal code changes. Great Work :)